include/exadg/incompressible_navier_stokes/postprocessor/output_generator.cpp

```diff
@@ -132,8 +131,13 @@ OutputGenerator<dim, Number>::setup(NavierStokesOperator const &    navier_stoke
   mapping                = &mapping_in;
   output_data            = output_data_in;
 
+  time_control.reinit(output_data_in.write_output,
+                      output_data_in.start_time,
+                      std::numeric_limits<double>::max(),
+                      output_data_in.interval_time,
+                      dealii::numbers::invalid_unsigned_int);
```

Comment:
we should only use ExaDG types here. Internally, `ExaDG::numbers::time_step` might potentially use `dealii::numbers::invalid_unsigned_int`.

Reply:
I did that already.